### DISCUSSION TITLE
Domain loco.rs expired

### DISCUSSION THREAD

**Discussion by JijoBose** [loco-rs/loco#1010] [General]
Looks like Domain loco.rs has expired or now an Unsupported TLD

**Comment by danmx** [comment]
It seems that it expired https://www.rnids.rs/en/whois/loco.rs

```
Domain name: loco.rs
Domain status: Expired https://www.rnids.rs/en/domain-name-status-codes#Expired
Registration date: 23.11.2023 12:38:21
Modification date: 14.12.2023 00:55:15
Expiration date: 23.11.2024 12:38:21
Confirmed: 14.12.2023 06:03:12
Registrar: Stanco d.o.o.
```

**Reply by jondot** [reply]
this was a strange experience as we got no notification from the registrar.
thanks! we've just renewed and monitoring closely to see when it is reinstated

**Comment by jondot** [comment]
back online,thanks!
